--- original.js	2018-08-17 20:04:22.000000000 +0100
+++ classy-mst.js	2018-08-17 20:06:06.000000000 +0100
@@ -81,9 +81,11 @@
     }).actions(function (self) {
         var result = {};
         var _loop_1 = function (name_3, value) {
-            result[name_3] = function () {
+             var func = function () {
                 return (value.apply(self, arguments));
             };
+            Object.defineProperty(func, 'name', {value: name + '.' + name_3, writable: false});
+            result[name_3] = func;
         };
         for (var _i = 0, actionList_1 = actionList; _i < actionList_1.length; _i++) {
             var _a = actionList_1[_i], name_3 = _a.name, value = _a.value;
