--- original.d.ts	2018-08-26 18:24:48.000000000 +0100
+++ jimp.d.ts	2018-08-26 19:08:47.000000000 +0100
@@ -501,7 +501,7 @@
                 reject: (reason: Error) => any,
                 ...args: T[]
             ) => any
-        );
+        ): any;
         static read(path: string): Promise<Jimp>;
         static read(image: Jimp): Promise<Jimp>;
         static read(data: Buffer): Promise<Jimp>;
@@ -533,12 +533,12 @@
             file: string,
             cb: Jimp.GenericCallback<Font, any, any>
         ): Promise<never>;
-        static measureText(font: Font, text: PrintableText);
+        static measureText(font: Font, text: PrintableText): any;
         static measureTextHeight(
             font: Font,
             text: PrintableText,
             maxWidth: number
-        );
+        ): any;
     }
 }
 
